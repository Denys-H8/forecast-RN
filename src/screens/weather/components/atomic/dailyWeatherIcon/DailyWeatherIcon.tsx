import React from 'react';

import Icon from 'react-native-vector-icons/Fontisto';
import constants from 'src/constants';

type TWeatherIcon = {
  weatherName: string;
};

const DailyWeatherIcon: React.FC<TWeatherIcon> = ({ weatherName }) => (
  <Icon name={weatherName} size={45} color={constants.SUB_COLOR} />
);

export default DailyWeatherIcon;
