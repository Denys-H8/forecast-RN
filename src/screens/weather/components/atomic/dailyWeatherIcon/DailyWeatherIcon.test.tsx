import React from 'react';
import { render } from '@testing-library/react-native';
import DailyWeatherIcon from './DailyWeatherIcon';

const min = 15;
const max = 27;

describe('DailyWeatherIcon ', () => {
  test('should render DailyWeatherIcon', () => {
    const { toJSON } = render(<DailyWeatherIcon />);

    expect(toJSON()).toMatchSnapshot();
  });
});
