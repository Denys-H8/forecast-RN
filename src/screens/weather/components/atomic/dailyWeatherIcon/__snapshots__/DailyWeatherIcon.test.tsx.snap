// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DailyWeatherIcon  should render DailyWeatherIcon 1`] = `
<Text
  allowFontScaling={false}
  style={
    Array [
      Object {
        "color": "#8B8EA5",
        "fontSize": 45,
      },
      undefined,
      Object {
        "fontFamily": "Fontisto",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      Object {},
    ]
  }
>
  
</Text>
`;
